src/utils.js

```diff
@@ -260,3 +260,16 @@ export function addMultipleEventListeners(_els, handler, _events) {
     });
   };
 }
+
+export const focusableElements = [
+  'a[href]',
+  'area[href]',
+  'input:not([disabled])',
+  'select:not([disabled])',
+  'textarea:not([disabled])',
+  'button:not([disabled])',
+  'object',
+  'embed',
+  '*[tabindex]:not(.modal)',
+  '*[contenteditable]',
```

Comment:
Probably also want `'audio[controls]'`, `'video[controls]'`, and to account for `contenteditable="false"` with `'[contenteditable]:not([contenteditable="false"])'`

Also, do you need the `*` in the selectors?

Reply:
Maybe not. Personal preference. Removed 👍